general/odoo_basics/add_user.rst

```diff
@@ -1,46 +1,141 @@
+   for more information.
+
+With the :doc:`Developer mode <../../general/developer_mode/activate>` activated, *User Types* can
+be selected.
+
+.. image:: media/user_type.png
+   :align: center
+   :height: 300
+   :alt: View of a user’s form in developer mode emphasizing the user type field in Odoo
+
+The *Portal* and *Public* options do not allow you to choose access rights. Members have specific
+ones (such as record rules and restricted menus), and they usually do not belong to the usual Odoo
+groups.
+
+.. _general/odoo_basics/add_user/access_rights:
+
+Access Rights in detail
+=======================
+
+Activate the :doc:`Developer mode <../../general/developer_mode/activate>` then go to
+:menuselection:`Settings --> Users & Companies --> Groups`.
+
+Groups
+~~~~~~
+
+| When choosing the groups the user can have access under
+  :ref:`Access Rights <general/odoo_basics/add_individual_user>`, details of the rules and
+  inheritances of that group are not shown, so this is when the menu *Groups* comes along. *Groups*
+  are created to define rules to models within an application.
+| Under *Users* have a list of the current ones. The ones with administrative rights are shown
+  in black.
+
+.. image:: media/groups_users.png
+   :align: center
+   :alt: View of a group’s form emphasizing the tab users in Odoo
+
+*Inherited* means that users added to this application group are automatically added to the
+following ones. In the example below, users who have access to the group *Administrator* of *Sales*,
+also have access to *Website/Restricted Editor* and *Sales/User: All Documents*.
+
+.. image:: media/groups_inherited.png
+   :align: center
+   :height: 330
+   :alt: View of a group’s form emphasizing the tab inherited in Odoo
 
-.. image:: media/add_user03.png
-    :align: center
+.. important::
+   Remember to always test the settings being changed in order to ensure that they are being applied
+   to the needed and right users.
+
+The *Menus* tab is where you define which menus (models) within the application the user can have
+access to.
+
+.. image:: media/groups_menus.png
+   :align: center
+   :height: 330
+   :alt: View of a group’s form emphasizing the tab menus in Odoo
+
+*Access Rights* rules are the first level of rights. The field is composed of the object name, which
+is the technical name given to a model. For each model, enable the following options as appropriate:
+
+- *Read*: the values of that object can just be read by the user.
+- *Write*: the values of that object can be edit by the user.
+- *Create*: values for that object can be created by the user.
+- *Delete*: the values of that object can be deleted by the user.
+
+.. image:: media/groups_access_rights.png
+   :align: center
+   :alt: View of a group’s form emphasizing the tab access rights in Odoo
+
+| As a second layer of editing and visibility rules, *Record Rules* can be formed. They overwrite,
+  or refine, the *Access Rights*.
```

Comment:
```suggestion
| As a second layer of editing and visibility rules, *Record Rules* can be formed. They overwrite
  or refine the *Access Rights*.
```

Reply:
I will keep the commas. If the sentence between the commas is removed, it does not impact the understanding of the whole. it is complementary.